pkg/gatewayserver/io/basicstationlns/basicstationlns_test.go

```diff
@@ -39,36 +39,41 @@ import (
 )
 
 var (
+	testServerAddress      = ":8100"
```

Comment:
- We're already in a test
- Do we need to declare these?
- You could listen on `:0` and use a port that's free for sure